src/clusterfuzz/stacktraces/__init__.py

```diff
@@ -875,6 +875,21 @@ def parse(self, stacktrace: str) -> CrashInfo:
             new_type='Out-of-memory',
             reset=True)
 
+      # Timeout/OOM detected by Centipede.
+      if self.detect_ooms_and_hangs:
```

Comment:
nit: just merge this if statement block with the one above. No need for a separate one.